plasmapy/analysis/swept_langmuir/helpers.py

```diff
@@ -0,0 +1,79 @@
+"""Helper functions for analyzing swept Langmuir traces."""
+__all__ = ["check_sweep"]
+
+import numpy as np
+
+
+def check_sweep(voltage: np.ndarray, current: np.ndarray) -> None:
+    """
+    Function for checking that the voltage and current arrays are properly
+    formatted for analysis by `plasmapy.analysis.swept_langmuir`.
+
+    Parameters
+    ----------
+    voltage: `numpy.ndarray`
+        1D `numpy.ndarray` representing the voltage of the swept Langmuir trace.
+        Voltage should be monotonically increasing.
```

Comment:
```suggestion
        Voltage should be monotonically increasing.  Assumes units of
        volts.
```
Assuming this does actually assume units of volts.

Reply:
I'm actually not enforcing units at this point.  I'm primarily doing this for speed reasons.  When analyzing data this routine will be called hundreds of times and I don't want to enforce unit validations every single time.  I'm thinking the `plasmapy.diagnostic` (aka `xarray`) version will have the appropriate unit handling associated with it.

Reply:
Yes and no,  there is no unit enforcement in this functionality.  In practice, the voltage array should be in volts and the current array should be in amperes.

I have added functionality to these checks that will strip units if a `u.Quantity` arrays is passed.

When it comes to the `xarray` version, I will enforce `volts` for the voltage array and `amperes` for the current array.